src/fetch/payouts.py

```diff
@@ -109,7 +117,11 @@ def from_series(cls, frame: Series) -> RewardAndPenaltyDatum:
             solver_name=frame["solver_name"],
             reward_target=Address(reward_target),
             payment_eth=int(frame["payment_eth"]),
-            slippage_eth=slippage,
+            # The simplest and most straightforward way to incorporate
+            # MEV kickback (owed to us) is as negative slippage.
+            # That is, Because this is an auxiliary term
+            # extrinsic to the competition reward mechanism
+            slippage_eth=slippage - mev_kickback,
```

Comment:
We may want to consider renaming this field to something like `payback_or_penalty`